src/WebJobs.Script/ExtensionBundle/ExtensionBundleManager.cs

```diff
@@ -263,7 +263,29 @@ internal static string FindBestVersionMatch(VersionRange versionRange, IEnumerab
                 return version;
             }).Where(v => v != null);
 
-            var matchingVersion = bundleVersions.OrderByDescending(version => version.Version).FirstOrDefault();
+            var latestBundleVersion = bundleVersions.OrderByDescending(version => version.Version).FirstOrDefault();
```

Comment:
Can we move all this new logic to a method? i.e. `ResolvePlatformReleaseChannelVersion` which returns the version